scania-digital-design-system/sdds-website
# fix(tab): padding correction

**Describe pull-request**  
Fixing spacing issue in the tab area

**How to test**  
Check what tab navigation looks like and if it is aligned with upper Hero/Header text.
PS: There is also a small update on the textfield, not done by me but I felt like including it too.
